src/main/java/Duke.java

```diff
@@ -1,10 +1,153 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + WELCOME_MESSAGE);
+        addingTasksToList();
+    }
+
+    public static final String BYE_MESSAGE = ""
+            + "____________________________________________________________\n"
```

Comment:
Can try changing all the quotes of underscores into a predefined constant, to ensure no magic literals kind of thing?